src/openms/include/OpenMS/ANALYSIS/NUXL/NuXLPresets.h

```diff
@@ -135,20 +136,18 @@ namespace OpenMS
     "X:-C5H8O4-H2S"
   };
 
-  static constexpr std::array<const char*, 13> modifications_RNA_UV_6SG_EXTENDED
+  static constexpr std::array<const char*, 11> modifications_RNA_UV_6SG_EXTENDED
   {   
     "X:",
     "X:-H2O",
     "X:-H3PO4",
     "X:-HPO3",
     "X:-H2S",
-    "X:+HPO3-H2S", // P1 artefact?
```

Comment:
Was added due to iTRAPP (4SU), thinking will work for 6SG as well
Did not find any single modification with this adduct.